src/component/AboutPage/Leadership/LeadershipComponenents/LeadershipCard/leadershipCard.jsx

```diff
@@ -17,20 +14,60 @@ export default function LeadershipCard({
     image = null,
     bio = "",
     emailUsername = "",
+    zIndex=0,
   } = {},
 }) {
   const [flipped, setFlipped] = useState(false);
-  const boldHeaderStyle = {
-    fontFamily: "Space Mono 700",
-  };
-
   const isBigScreen = !window.matchMedia("(max-device-width: 650px)").matches;
 
-  const detailsRowJustify = title ? "between" : "end";
+  const yPad = isBigScreen ? ".5vh" : ".5vw";
```

Comment:
This is fine, one thing to note is that this won't be responsive, and doesn't work with the actual screen width

Reply:
^this being the .matchMedia(max-device-width)

Reply:
This is for mobile, the person before named it isBigScreen, but this is just so I can change the button size on mobile vs desktop. It shouldn't resize by width on desktop.

Reply:
Probably can change it to isDesktop or something along those lines for better clarification